SUMMIT2020-70: Initial opensync vendor code for rpi

vdkmai opened the pull request:
Reason for change: Bring up OpenSync on Rpi
Test Procedure: N/A
Risks: Low

Signed-off-by: Khoa Mai <vdkmai@humaxdigital.com>

vdkmai committed:
SUMMIT2020-70: Initial opensync vendor code for rpi

Reason for change: Bring up OpenSync on Rpi
Test Procedure: N/A
Risks: Low

Signed-off-by: Khoa Mai <vdkmai@humaxdigital.com>

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/rdkcentral/opensync-vendor-rdk-rpi?pullRequest=2) <br/>All committers have signed the CLA.

rdkcmf-jenkins commented:
## Copyright scan failure
 **Commit:** 9c51f8ab946ab31651fc7670e0709e98b3ae175f
**Report detail:**  https://gist.github.com/88fb66bd21f1534491d480eea398d3b5

rdkcmf-jenkins commented:
## Blackduck scan failure details

**Summary**: 0 **violation**s, 0 files pending approval, 7 files pending identification.
* Protex Server Path: /home/blackduck/github/opensync-vendor-rdk-rpi/2/rdkcentral/opensync-vendor-rdk-rpi

* Commit: 9c51f8ab946ab31651fc7670e0709e98b3ae175f

 **Report detail:** [gist](https://gist.github.com/053e86d4f54b2d7a0dc5b9a38a785948)

vdkmai committed:
SUMMIT2020-70: Fix opensync build error on extender profile

Reason for change: Bring up OpenSync Extender on Rpi
Test Procedure: Tested with local build
Risks: Low

Signed-off-by: Khoa Mai <vdkmai@humaxdigital.com>
